build.gradle.kts

```diff
@@ -132,6 +138,7 @@ val copyMavenScripts by tasks.registering(Copy::class) {
 
     // local variable required for configuration cache compatibility
     // https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:not_yet_implemented:accessing_top_level_at_execution
+    val buildScanSummaryVersion = buildScanSummaryVersion
     val releaseVersion = releaseVersion
     inputs.property("project.version", releaseVersion)
```

Comment:
Do we also need to declare an input property for the buildScanSummaryVersion ?

Reply:
See https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/487#discussion_r1303353568